src/System.Drawing.Common/src/System/Drawing/Font.Unix.cs

```diff
@@ -576,117 +152,20 @@ public static Font FromLogFont(object lf, IntPtr hdc)
             return new Font(newObject, "Microsoft Sans Serif", FontStyle.Regular, 10);
         }
 
-        public float GetHeight()
-        {
-            return GetHeight(Graphics.systemDpiY);
-        }
-
         public static Font FromLogFont(object lf)
```

Comment:
I don't understand this change.  In the new version, the `lf` argument is never used.  How is that possibly correct?

Reply:
Accidental. This has now been fixed - I verified this by testing it manually, as the tests are not supported on Unix

Reply:
> as the tests are not supported on Unix

Which tests aren't supported on Unix?  I'm nervous about such a large refactoring with insufficient test coverage.